CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+## [2.1.0] - 2018-12-06
+### Added
+- Use wait_until_available to reduce task runs ([BOLT-956](https://tickets.puppetlabs.com/browse/BOLT-956))
+- Add bash and powershell implementations of reboot task ([BOLT-459](https://tickets.puppetlabs.com/browse/BOLT-459))
+- Add wait function for Puppet Plans ([MODULES-8046](https://tickets.puppetlabs.com/browse/MODULES-8046))
```

Comment:
This was reverted. Should we still include it? Is it always going to be re-added by automatic changelog generation?

Reply:
Whoops yes.  I missed the revert.  Fixed.